stable/coredns/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.rbac.create }}
```

Comment:
See https://github.com/helm/helm/blob/master/docs/chart_best_practices/rbac.md for how to create serviceaccount.
I think the PR is pretty ready if this can be fixed.
Also see the mergeconflict for Chart.yaml

Reply:
Thanks, this was hidden under the rbac.yaml before but I can fix it.

Reply:
Fixed now.